assets/js/selecting.js

```diff
@@ -19,33 +19,116 @@ function sslothable(s, n) {
+                window.location.href = "installing-boot9strap-(soundhax-sp)";
+            }
+            else if (sslothable(minor.value, nver.value)) { //check for versions that are not cartupdated, cartupdated consoles cannot access the browser, see troubleshooting for solution
+                window.location.href = "installing-boot9strap-(ssloth-browser)";
+            }
+            else { //seedminer does still work for the latest version on E/U/J/K/T/C, but can only be chained on E/U/J/K/T
+                window.location.href = "seedminer";
+            }
+        } 
+        if (region.value === "J") { //japan stuff
+            if (major.value < 11 || minor.value < 4) { //soundhax works on all consoles for 1.0-11.3
+                window.location.href = "installing-boot9strap-(soundhax)";
+           }
+            else if (nver.value < 36) { //if the nver is below 36 its a cartupdate from below 11.3, which makes soundhax possible
+                window.location.href = "installing-boot9strap-(soundhax-sp)";
+            }
+            else if (sslothable(minor.value, nver.value)) { //check for versions that are not cartupdated, cartupdated consoles cannot access the browser, see troubleshooting for solution
+                window.location.href = "installing-boot9strap-(ssloth-browser)";
+            }
+            else { //seedminer does still work for the latest version on E/U/J/K/T/C, but can only be chained on E/U/J/K/T
+                window.location.href = "seedminer";
+            }
+        } 
+        if (region.value === "K") { //korea stuff
+            if (major.value < 11 || minor.value < 4) { //soundhax works on all consoles for 1.0-11.3
+                window.location.href = "installing-boot9strap-(soundhax)";
+            }
+            else if (nver.value < 33) { //if the nver is below 36 its a cartupdate from below 11.3, which makes soundhax possible
+                window.location.href = "installing-boot9strap-(soundhax-sp-kor)";
+            }
+            else if (sslothablekor(minor.value, nver.value)) { //check for versions that are not cartupdated, cartupdated consoles cannot access the browser, see troubleshooting for solution
+                window.location.href = "installing-boot9strap-(ssloth-browser)";
+            }
+            else { //seedminer does still work for the latest version on E/U/J/K/T/C, but can only be chained on E/U/J/K/T
+                window.location.href = "seedminer";
+            }
+        } 
+        if (region.value === "T") { //taiwan stuff
+            if (major.value < 11 || minor.value < 4) { //soundhax works on all consoles for 1.0-11.3
+                window.location.href = "installing-boot9strap-(soundhax)";
+            }
+            else if (nver.value < 36) { //if the nver is below 36 its a cartupdate from below 11.3, which makes soundhax possible
```

Comment:
although not really related to Korean console, but since this pr also fix TWN/CHN issue, this should changed to 24 (hope this is correct) and also with soundhax-sp-twn page, or commented out temporarily before confirmed and actually fixed.